src/ceph-volume/ceph_volume/tests/util/test_disk.py

```diff
@@ -250,138 +166,88 @@ def test_float(self):
 class TestGetDevices(object):
 
     def setup_paths(self, tmpdir):
-        paths = []
-        for directory in ['block', 'dev', 'mapper']:
-            path = os.path.join(str(tmpdir), directory)
-            paths.append(path)
-            os.makedirs(path)
-        return paths
-
-    def test_no_devices_are_found(self, tmpdir):
-        result = disk.get_devices(
-            _sys_block_path=str(tmpdir),
-            _dev_path=str(tmpdir),
-            _mapper_path=str(tmpdir))
-        assert result == {}
+        path = os.path.join(str(tmpdir), 'block')
```

Comment:
why is this not looking at dev or mapper paths?

Reply:
Because is doesn't need to. Do you see anything wrong with the behaviour?